tests/schemas/invalid/test_invalid_validation_sum_against_total_dynamic_answers.json

```diff
@@ -0,0 +1,253 @@
+                      "placeholder": "supermarket_name",
+                      "transforms": [
+                        {
+                          "arguments": {
+                            "delimiter": " ",
+                            "list_to_concatenate": [
+                              {
+                                "source": "answers",
+                                "identifier": "supermarket-name"
+                              }
+                            ]
+                          },
+                          "transform": "concatenate_list"
+                        }
+                      ]
+                    }
+                  ]
+                }
+              }
+            },
+            {
+              "type": "Question",
+              "id": "dynamic-answer-only",
+              "skip_conditions": {
+                "when": {
+                  "==": [
+                    {
+                      "count": [
+                        {
+                          "source": "list",
+                          "identifier": "supermarkets"
+                        }
+                      ]
+                    },
+                    0
+                  ]
+                }
+              },
+              "question": {
+                "guidance": {
+                  "contents": [
+                    {
+                      "description": "Answers are validated against total amount from previous question."
+                    }
+                  ]
+                },
+                "answers": [
+                  {
+                    "label": {
+                      "text": "Percentage of shopping elsewhere",
+                      "placeholders": [
+                        {
+                          "placeholder": "transformed_value",
+                          "value": {
+                            "source": "answers",
+                            "identifier": "supermarket-name"
+                          }
+                        }
+                      ]
+                    },
+                    "id": "percentage-of-shopping-elsewhere",
+                    "mandatory": false,
+                    "type": "Percentage",
+                    "maximum": {
+                      "value": 100
+                    },
+                    "decimal_places": 0
+                  }
+                ],
+                "warning": "These answers must add up to the total provided in the previous question",
+                "calculations": [
+                  {
+                    "calculation_type": "sum",
+                    "value": 1000,
+                    "answers_to_calculate": [
+                      "percentage-of-shopping-elsewhere"
+                    ],
+                    "conditions": ["equals"]
+                  }
+                ],
+                "id": "dynamic-answer-only-question",
+                "title": "How much do you spend at each of the following supermarket?",
```

Comment:
fails as expected if I put this in the valid schemas folder but I think you need a test that loads this schema and asserts that you get the correct Answers too short message when on this block
at the moment when I run unit tests the coverage report doesn't include the new type of error

Reply:
Added a test now, thanks.